membership-attribute-service/test/services/PaymentFailureAlerterTest.scala

```diff
@@ -77,19 +87,19 @@ class PaymentFailureAlerterTest(implicit ee: ExecutionEnv)  extends Specificatio
         result must be_==(false).await
       }
 
-      "return true for for a member with a failed payment and an invoice in the last 27 days" in {
+      "return true for a member with a failed payment and an invoice in the last 27 days" in {
         val result = PaymentFailureAlerter.alertAvailableFor(accountObjectWithBalance, membership, paymentMethodResponseRecentFailure)
 
         result must be_==(true).await
       }
 
-      "return true for for a non-membership sub with a failed payment in the last 27 days too" in {
-        val result = PaymentFailureAlerter.alertAvailableFor(accountObjectWithBalance, digipack, paymentMethodResponseRecentFailure)
```

Comment:
This test will no longer pass because it now doesn't true for subs that are not contributions or memberships